include/odp/api/spec/ipsec.h

```diff
@@ -762,6 +762,37 @@ typedef struct odp_ipsec_sa_param_t {
 
 } odp_ipsec_sa_param_t;
 
+/**
+ * Retrieve IPSEC SA information
+ */
+typedef struct odp_ipsec_sa_info_t {
+	/** IPSEC Security Association (SA) parameters */
+	odp_ipsec_sa_param_t param;
```

Comment:
As discussed last week, current pattern in ODP info APIs is that if there's param structure included, it is a copy of parameters from application (from sa create). Additional info fields represent data that is dynamic/implementation picked values.

Params may be included here or not (no strict opinion either way), since application should be able to remember param values used in sa creation phase.

Reply:
Agreed that the param structure in sa_info should be a copy of parameters provided by the application.
In V2, I am updating the description of the field to:
`       /** Copy of IPSEC Security Association (SA) parameters */ `

Let me know if this would be enough.